client/src/store/stores/main.ts

```diff
@@ -185,6 +186,20 @@ export class Store {
       reaction.dispose();
     });
 
+    autorun(reaction => {
```

Comment:
There is no need to make a reaction for it. Take a look at how namespace is restored in constuctor, L65.
Btw, reaction on L169 is redundant and should be removed ><